tests/ImageSharp.Tests/ImageSharp.Tests.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="CoreCompat.System.Drawing" Version="1.0.0-beta006" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
```

Comment:
Do we need this?

Reply:
This enables unit tests execution in JetBrains Rider, but it's not necessary.

Reply:
Let's keep it then. I want everyone to be able to test the project in their desired IDE.